adapters/saml/undertow/pom.xml

```diff
@@ -32,12 +32,6 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.jboss.logging</groupId>
```

Comment:
Yes, it was prerequisite to maven project to be valid. Should I update PR and drop this commit from it?